packages/colorpickers/src/elements/ColorPicker/ColorWell.tsx

```diff
@@ -22,37 +22,52 @@ interface IColorWellProps {
   hue: number;
   saturation: number;
   lightness: number;
-  onChange?: (hsv: IHSVColor, event: React.MouseEvent<any>) => void;
+  onChange?: (hsv: IHSVColor, event: MouseEvent) => void;
 }
 
 export const ColorWell: React.FC<IColorWellProps> = ({ hue, saturation, lightness, onChange }) => {
   const { rtl } = useContext(ThemeContext);
   const container = useRef<HTMLDivElement>(null);
   const hsv = hsl2hsv(hue, saturation, lightness);
 
-  const throttledChange = throttle(e => {
-    if (container.current) {
-      const nextHsv = calculateNextHsv(e, hsv, container.current, rtl);
+  const throttledChange = useMemo(() => {
+    /* eslint-disable-next-line @typescript-eslint/no-shadow */
+    return throttle((e: MouseEvent, hsv: IHSVColor, rtl: boolean) => {
+      if (container.current) {
+        const nextHsv = calculateNextHsv(e, hsv, container.current, rtl);
 
-      onChange && onChange(nextHsv, e);
-    }
-  }, 50);
+        onChange && onChange(nextHsv, e);
+      }
+    }, 50);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Is this to remove the need for the `onChange` dependency? Why is this necessary?